types/default_override_test.conf

```diff
@@ -0,0 +1,11 @@
+[storage]
+
+# Default Storage Driver
+driver = ""
+
+# Primary Read/Write location of container storage
+graphroot = "environment_override_graphroot"
+
+# Storage path for rootless users
+#
+rootless_storage_path = "environment_override_rootless_storage_path"
```

Comment:
Is this file meant to be used in a test?

Reply:
Yes it was, but calling the `DefaultStoreOptions` function from the tests seems to break all tests.
Can you help with a better way of testing this? Else i can remove it

Reply:
I would imagine a test that would use [os.Setenv()](https://golang.org/pkg/os/#Setenv) and query for the expected values of the loaded storage conf.